docs/business.yaml

```diff
@@ -3740,6 +3680,78 @@ components:
                 title: Path of the property in json
       description: '***Item not used*** What will be changed in the correction filing.'
       type: object
+    Directors:
+      title: Directors Schema
+      type: object
```

Comment:
think the schema should just represent on director.  and anything using it can define an array of directors referencing this schema if required.

Reply:
Thank you, Argus. 
Yes, you're right. Will update it.